src/Components/Publishing/Header/ClassicHeader.tsx

```diff
@@ -19,7 +19,7 @@ export const ClassicHeader: React.SFC<ClassicHeaderProps> = props => {
       <Title>{title}</Title>
       {leadParagraph}
       <AuthorDateClassic
-        authors={article.contributing_authors}
+        authors={article.authors}
```

Comment:
We'll still need to support contributing_authors for Classic articles since the concept of `authors` only exists for Editorial.

Reply:
👍  will update this now